lib/active_admin.rb

```diff
@@ -128,3 +130,4 @@ def after_load(&block)
 # Require ORM-specific plugins
 require 'active_admin/orm/active_record' if defined? ActiveRecord
 require 'active_admin/orm/mongoid'       if defined? Mongoid
+require 'active_admin/error_pages/error_pages'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.